builder-base/Makefile

```diff
@@ -25,6 +25,7 @@ local-images:
 		--local dockerfile=./ \
 		--local context=. \
 		--output type=tar,dest=/tmp/builder-base/output.tar
+	./update_base_image.sh --dry-run
 
 .PHONY: images
 images:
```

Comment:
We didn't have the sleep here? I wonder why

Reply:
Guess the buildkit daemon had started by then, maybe we got lucky. Added the sleep for both presubmits and postsubmits